tools/e2e-tests/apps/symlink-monorepo/app/client/main.html

```
@@ -0,0 +1,8 @@
+<head>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)